flink-connectors/flink-connector-rabbitmq/src/main/java/org/apache/flink/streaming/connectors/rabbitmq/common/RMQConnectionConfig.java

```diff
@@ -343,6 +367,8 @@ public ConnectionFactory getConnectionFactory()
         // basicQos options for consumers
         private Integer prefetchCount;
 
+        private Integer deliveryTimeout;
```

Comment:
IMO, the type of `deliveryTimeout` is `Long`.

Reply:
Agree, it sounds more intuitive, thanks. Changed in 54b08d3